modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/model/listener/DDMStructureModelListener.java

```diff
@@ -55,13 +54,8 @@ public void onBeforeUpdate(
 		JournalServiceConfiguration journalServiceConfiguration =
 			_getJournalServiceConfiguration(ddmStructure.getCompanyId());
 
-		if ((FeatureFlagManagerUtil.isEnabled(
-				ddmStructure.getCompanyId(), "LPD-25799") &&
-			 ((journalServiceConfiguration == null) ||
-			  journalServiceConfiguration.autogenerateDDMStructureKey())) ||
-			(!FeatureFlagManagerUtil.isEnabled(
-				ddmStructure.getCompanyId(), "LPD-25799") &&
-			 _journalWebConfiguration.autogenerateDDMStructureKey()) ||
+		if ((journalServiceConfiguration == null) ||
```

Comment:
And  I think this condition make not sense: 

```
(journalServiceConfiguration == null) ||
			journalServiceConfiguration.autogenerateDDMStructureKey()
```

because we will throw an exception when journalServiceConfiguration is null.

It should be:

```
(journalServiceConfiguration != null)  &&
			journalServiceConfiguration.autogenerateDDMStructureKey()
```

It's my mistake. What do you think?

Reply:
Changes applied @Mikellorza, thanks!